### PR TITLE
fix(1091): add aggregation field to scan schema

### PR DESCRIPTION
**Pull request by InderH**
## Context

Need to allow "aggregationField" in scan config so aggregation queries can be made.

## Objective

This PR adds "aggregationField" to SCHEMA_SCAN so that scan configs that include aggregationField pass validation.

## References

https://github.com/screwdriver-cd/screwdriver/issues/1091

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: migrations/20200203-add-column_templateId_to_jobs.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ module.exports = {
         await queryInterface.sequelize.transaction(async (transaction) => {
             await queryInterface.addColumn(table, 'templateId', {
                 type: Sequelize.DOUBLE }, { transaction });
```
Comment (parthasl) [inline migrations/20200203-add-column_templateId_to_jobs.js]: change templateId to Integer

File: migrations/20200204-add-column_templateId_to_builds.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ module.exports = {
         await queryInterface.sequelize.transaction(async (transaction) => {
             await queryInterface.addColumn(table, 'templateId', {
                 type: Sequelize.DOUBLE }, { transaction });
```
Comment (parthasl) [inline migrations/20200204-add-column_templateId_to_builds.js]: same templateId to Integer

File: migrations/20200203-add-column_templateId_to_jobs.js (hunk 1 above)
Comment (InderH) [reply migrations/20200203-add-column_templateId_to_jobs.js]: updated, thanks

File: migrations/20200204-add-column_templateId_to_builds.js (hunk 2 above)
Comment (InderH) [reply migrations/20200204-add-column_templateId_to_builds.js]: updated, thanks

**Pull request merged**